mkdocs.yml

```
@@ -1,10 +1,31 @@
-  - Home: index.md
-  - DDD for Pytest Fixer Concepts and Implementation Guide: DDD-for-pytest-fixer-Concepts-and-Implementation-Guide.md
-  - Pytest Fixer TDD Blueprint: Pytest-Fixer-TDD-Blueprint.md
-  - Repository Structure and Architecture Design Proposal: Repository-Structure-and-Architecture-Design-Proposal.md
-  - User & Developer Guide: pytest-fixer-User-&-Developer-Guide.md
+site_name: Pytest Fixer
 theme: readthedocs
+
+nav:
+  - 'Home': 'index.md'
+  - 'User Guide':
+    - 'Installation': 'user-guide/01-installation.md'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)